be/src/service/staros_worker.cpp

```diff
@@ -16,20 +16,14 @@ namespace starrocks {
 
 namespace fslib = staros::starlet::fslib;
 
-std::ostream& operator<<(std::ostream& os, const staros::starlet::ShardInfo& shard) {
-    return os << "Shard{.id=" << shard.id << " .uri=" << shard.obj_store_info.s3_obj_store.uri << "}";
-}
-
 absl::Status StarOSWorker::add_shard(const ShardInfo& shard) {
     std::unique_lock l(_mtx);
-    LOG(INFO) << "Adding " << shard;
```

Comment:
this line is helping troubleshooting a lot, may still need to keep it

Reply:
starlet has already log this infomation